src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
just give one common class for all block, for ex. `stars`, and then you can go like this:
```css
.stars {
  display: flex;
}
```
and no need to set flex-direction: row, because it is row by default